src/lib/faqItem.svelte

```diff
@@ -11,12 +11,12 @@
 		<!-- Expand/collapse question button -->
 		<button
 			type="button"
-			class="text-left w-full flex justify-between items-start text-v-black"
+			class="my-4 text-left w-full flex justify-between items-start text-v-black"
 			aria-controls="faq-0"
 			aria-expanded="false"
 			on:click={() => (show = !show)}
 		>
-			<h4 class="mb-0">{question}</h4>
+			<p class="mb-0 text-xl">{question}</p>
```

Comment:
Same as comment further down, we want to keep using `h1-4` elements for consistency across the site.